Xamarin.Forms.Core/Label.cs

```diff
@@ -58,6 +58,8 @@ public class Label : View, IFontElement, ITextElement, ITextAlignmentElement, IE
 		public static readonly BindableProperty LineBreakModeProperty = BindableProperty.Create(nameof(LineBreakMode), typeof(LineBreakMode), typeof(Label), LineBreakMode.WordWrap,
 			propertyChanged: (bindable, oldvalue, newvalue) => ((Label)bindable).InvalidateMeasureInternal(InvalidationTrigger.MeasureChanged));
 
+		public static readonly BindableProperty LineHeightProperty = BindableProperty.Create("LineHeight", typeof(double), typeof(Label), -1.0);
```

Comment:
- use `nameof(LineHeight)`
- you need to InvalidateMeasureInternal on line-height change
- I prefer `-1d` to `-1.0`, but that's probably just me